cob_hardware_config/raw3-1/config/arm_cartesian_controller.yaml

```diff
@@ -0,0 +1,79 @@
+## Cartesian controller parameters
+chain_base_link: arm_base_link
+chain_tip_link: arm_ee_link
+root_frame: odom_combined
+
+# twist controller parameters
+twist_controller:
+  controller_interface: cob_twist_controller/ControllerInterfaceVelocity
+  integrator_smoothing: 0.2
+
+  # damping parameters
+  numerical_filtering: false
+  damping_method: 2    #NO 0, CONSTANT 1, MANIPULABILITY 2, LeastSingularValue 3
+  damping_factor: 0.01
+  lambda_max: 0.1
+  w_threshold: 0.005
+  beta: 0.005
+  eps_damping: 0.003
+  eps_truncation: 0.001
+
+  # constraints
+  solver: 3    #Default 0, WLN 1, GPM 2, StackOfTask 3, Task2nd 4
+  priority: 500    #Prio for main task
+  k_H: 1.0    #Self-motion factor for GPM
+
+  # JLA
+  constraint_jla: 1    #OFF 0, ENABLED 1, MID 1, INEQ 2
+  priority_jla: 50
+  k_H_jla: -1.0
+  activation_threshold_jla: 10.0    #in [%]
+  damping_jla: 0.000001
+
+  # CA
+  constraint_ca: 1    #OFF 0, ENABLED 1
+  priority_ca: 100
+  k_H_ca: -1.0
+  activation_threshold_ca: 0.1    #in [m]
+  damping_ca: 0.000001
+
+  # limit enforcing
+  keep_direction: true
+  enforce_pos_limits: true
+  enforce_vel_limits: true
+  enforce_acc_limits: false
+  limits_tolerance: 5.0    #for position limits [deg]
+  #limits_acc: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]    #[rad/s2]
```

Comment:
please update according to https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/cob4-5/config/arm_left_cartesian_controller.yaml#L40-L49